transitions/accept_patient_reports.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
         var self = module.exports;
         return Boolean(
             doc &&
+            doc.type === 'data_record' &&
```

Comment:
We don't actually need this check because we check for the existence of a form, which is only on a `data_record` anyway. I'm adding it here to be explicit.